mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,14 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+// makeGubernatorLink returns a URL to view the build results in a GCS path.
+//
+// Gubernator is a simple frontend that reads test result buckets to improve
+// test triaging. Its source code is in kubernetes/test-infra/gubernator
+func makeGubernatorLink(id string) string {
```

Comment:
Nits addressed.

On Thu, Jun 2, 2016 at 1:10 PM, Daniel Smith <notifications@github.com>
wrote:

> In mungegithub/mungers/flake-manager.go
> <https://github.com/kubernetes/contrib/pull/1112#discussion_r65609670>:
>
> > @@ -127,6 +127,14 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
> >  	return true
> >  }
> >
> > +// makeGubernatorLink returns a URL to view the build results in a GCS path.
> > +//
> > +// Gubernator is a simple frontend that reads test result buckets to improve
> > +// test triaging. Its source code is in kubernetes/test-infra/gubernator
> > +func makeGubernatorLink(id string) string {
>
> probably going to merge & push this shortly because I want this fix, but
> please come up with a better variable name than ID, and explain what its
> contents should be.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1112/files/d95e3114a4651a97981cfe47414dd5f4299fa976#r65609670>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAM7nKaQfJsAvSLLXFwlveblddnISQtWks5qHzjSgaJpZM4IrErK>
> .
>